src/components/ApiData.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './ApiData.css';
+
+const FetchQuote = () => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [response, setResponse] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setLoading(true);
+      try {
+        const category = 'learning';
+        const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+        const ApiKey = 'pN7C6QCqr4X5uH9a5SJmXg==IHJMOO4hHQc5s0hK';
+        const headers = {
+          'Content-Type': 'application/json',
+          'X-Api-Key': ApiKey,
+        };
+
+        const response = await fetch(url, { headers });
+        const data = await response.json();
+        setResponse(data);
+      } catch (error) {
+        setError(true);
+      }
+      setLoading(false);
+    };
+
+    fetchData();
+  }, []);
+
+  if (loading) return <h2 className="loading">Loading...</h2>;
+  if (error) return <h2 className="error">An error occurred!</h2>;
+
+  return (
+    <div className="quotes">
+      {response.map((quote) => (
+        <div key={quote.id}>
+          {quote.quote}
+          <p className="author">{quote.author}</p>
+        </div>
+      ))}
+    </div>
+  );
+};
+
+export default FetchQuote;
```

Comment:
Just so you know, currently, when your app loads, no quote is displayed. Not to worry, it is due to a simple miss, and it happens to the best of us 🤷🏽‍♂️. Kindly ensure the `ApiData` component should be imported into your app.js:
![Screenshot from 2023-07-13 17-56-06](https://github.com/Dorcas126/math-magicians/assets/44624138/8ed772ec-213f-4cec-b9db-9a2e7ff3b069)